samples/stocks/command.js

```diff
@@ -0,0 +1,31 @@
+const Superagent = require('superagent');
+
+/**
+* This is the implementation of your Fusebot command.
+* You can run it from Slack with `/fusebot stocks ...`.
+* Fusebot is powered by the Fusebit integration platform, https://fusebit.io/
+* 
+* Docs and samples are at https://github.com/fusebit/fusebot
+* You can talk to a fellow developer on Slack at https://fusebit.io/contact
+* Or tell us what you think with `/fusebot feedback {your-comments}` 
+*
+* Now let's build something already. 
+
+* @param ctx {FusebotContext}
+*/
+module.exports = async (ctx) => {
+  if (ctx.body.args.length === 0 || ctx.body.args[0] === 'help') {
+    return help(ctx)
+  }
+  const stockSymbol = ctx.body.args[0]
+  const results = await Superagent.get(`https://www.alphavantage.co/query\?function\=GLOBAL_QUOTE\&symbol\=${stockSymbol}\&apikey\=${ctx.configuration.API_KEY}`)
+  await ctx.client.send(`${stockSymbol}'s current price is ${results.body['Global Quote']['05. price']}.`)
```

Comment:
What is that `05. price` property?

Reply:
the API have some interesting response :/. its the current price of the stock.

Reply:
Here's an example from their Quote Endpoint: https://www.alphavantage.co/documentation/#latestprice

As @matthewzhaocc said, the key names aren't what you expect: https://www.alphavantage.co/query?function=GLOBAL_QUOTE&symbol=IBM&apikey=demo

They look human-readable names/values for the keys rather than API-friendly names.

@matthewzhaocc maybe link to the [docs](https://www.alphavantage.co/documentation/#latestprice) for this endpoint in the comments? Just a suggestion.

Reply:
good idea, thanks!